src/docs/platforms/javascript/angular.mdx

```diff
@@ -5,150 +5,253 @@ redirect_from:
-      console.error(extractedError);
-    }
-
-    // Optionally show user dialog to provide details on what happened.
-    Sentry.showReportDialog({ eventId });
-  }
-}
+```javascript
+import { NgModule, ErrorHandler } from '@angular/core';
+import { createErrorHandler } from '@sentry/angular';
 
 @NgModule({
-  declarations: [AppComponent],
-  imports: [BrowserModule],
-  providers: [{ provide: ErrorHandler, useClass: SentryErrorHandler }],
-  bootstrap: [AppComponent],
+  // ...
+  providers: [
+    {
+      provide: ErrorHandler,
+      useValue: createErrorHandler({
+        showDialog: true,
+      }),
+    },
+  ],
+  // ...
 })
 export class AppModule {}
 ```
 
-When using your own `ErrorHandler`, make sure that whenever you use `HttpInterceptor` alongside it,
-the interceptor doesn't modify the error captured originally.
-The same goes for writing your own API services with built-in `http` methods.
+Additionally, `createErrorHandler` accepts a set of options that allows you to configure its behavior. For more details see the `ErrorHandlerOptions` interface in [our repository](https://github.com/getsentry/sentry-javascript/blob/master/packages/angular/src/errorhandler.ts).
 
-For example, the service below makes it impossible for the SDK to extract the correct data, because the full, original error `e` is not propagated.
+## Integrating Tracing
 
-```js
-export class ApiService {
-  constructor(http) {
-    this.http = http;
-  }
+`@sentry/angular` exports a Trace Service, Directive, and Decorators that leverages the `@sentry/tracing`, Sentry's Tracing integration, to add Angular-related spans to transactions. The service itself tracks route changes and durations, where directive and decorators are tracking component initializations.
 
-  formatErrors(e) {
-    console.log("Captured service error");
-    return throwError(e.error);
-  }
+**Note:** If the Tracing integration is not enabled, this functionality will not work. 
 
-  get(path) {
-    return this.http.get(path).pipe(catchError(this.formatErrors));
-  }
-}
-```
+### Install
 
-Instead, make sure that you always rethrow or directly pass the original error. For example:
+Install a Trace Service with these three steps:
 
-```js
-export class ApiService {
-  constructor(http) {
-    this.http = http;
+1. Register and configure the Tracing integration, including custom Angular routing instrumentation:
+
+  ```javascript
+  import { init, routingInstrumentation } from '@sentry/angular';
+  import { Integrations as TracingIntegrations } from '@sentry/tracing';
+  
+  init({
+    dsn: '___PUBLIC_DSN___',
+    integrations: [
+      new TracingIntegrations.BrowserTracing({
+        tracingOrigins: ['localhost', 'https://yourserver.io/api'],
+        routingInstrumentation: routingInstrumentation,
+      }),
+    ],
+    tracesSampleRate: 1,
+  });
+  ```
+
+2. Register `SentryTrace` as a provider in Angular's DI system, with a `Router` as its dependency:
```

Comment:
I don't see `SentryTrace` in the following code example, should this read "Register Sentry's Tracing integration as a provider..."?

Reply:
It should be `TraceService` instead. Fixed.